client/my-sites/stats/annual-site-stats/index.js

```diff
@@ -177,7 +177,7 @@ class AnnualSiteStats extends Component {
 				{ isWidget && (
 					<SectionHeader
 						href={ viewAllLink }
-						label={ translate( 'Annual Site Stats', { args: [ currentYear ] } ) }
+						label={ translate( 'Annual site stats', { args: [ currentYear ] } ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.